rules_bot.py

```diff
@@ -161,6 +163,12 @@ def main() -> None:
     # Tag hints - works with regex
     application.add_handler(MessageHandler(TagHintFilter(), tag_hint))
 
+    # Warn user who shared a bot's token
+    application.add_handler(CommandHandler("token", command_token_warning))
```

Comment:
please also add the `/token` command to wherever we call `set_my_commands` :D it should be available for both admins and normal group members